src/isar/state_machine/states/monitor.py

```diff
@@ -126,6 +121,29 @@ def _run(self) -> None:
                     transition = self.state_machine.step_finished  # type: ignore
                     break
             else:
+                if (
```

Comment:
The only change is that it assumes that missionwise isar implementations can report step status, but I can add a config variable (in the CAPABILITIES list) which states whether a robot is able to report step status.

Reply:
Then we need to add in the capabilities of all the other robots yeah? And make sure our deployed robots have updated config in our environments?

Reply:
Not necessarily, it would only need to be updated for robots that are missionwise but also need to report step status. Unless we want to require that any robot which is capable of reporting step status has it, even those which are stepwise anyways

Reply:
Could be a little confusing as the stepwise robots also report step status

Reply:
It might be easiest to assume that any robot can report step status. This seems reasonable as we anyways expect other telemetry.

Reply:
I am changing it to a setting, which is only used for missionwise robots.